anchor/common/operator_key/src/encrypted.rs

```diff
@@ -92,8 +95,8 @@ impl EncryptedKey {
 
     /// Decrypt the private key from the keystore.
     ///
-    /// If the pubkey was provided along the encrypted key in a "pubKey" attribute, it is verified
-    /// whether the encrypted key matches the public key.
+    /// If the pubkey was provided along the encrypted key in a "pubkey" or "pubKey" attribute, it
```

Comment:
Any reason not to say that pubKey is legacy and supported for backward compatibility?